cvmfs/catalog_mgr.h

```diff
@@ -183,6 +183,7 @@ class AbstractCatalogManager : public SingleCopy {
   uint64_t GetRevision() const;
   bool GetVolatileFlag() const;
   uint64_t GetTTL() const;
+  bool GetVOMSAuthz(std::string&) const;
```

Comment:
I would like this to be fixed in the "tech preview" and gather input from VOs whether they would want different authentication for nested catalogs.

Seems limited value since it's a bit impractical to make the symmetric narrowing of authorization in the HTTPS server.